modes/ssh_chat.py

```
@@ -804,3 +807,341 @@ def _handle_queue(self) -> None:
+            return
+
+        if not args:
+            print(f"{Colors.INFO}Usage:{Colors.RESET}")
+            print("  /task <title>           - Create a new task")
+            print("  /task <id>              - Show task details")
+            print("  /task <title> !high     - Create high-priority task")
+            print("  /task <title> #tag      - Create task with tag")
+            return
+
+        # Check if it's a task ID (8 or 36 characters UUID)
+        if len(args) in [8, 36] and "-" in args or args.count("-") >= 3:
```

Ruff (0.14.14)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)